### PR TITLE
Refactor ProductHeroV2 layout

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Colocate `ProductHero` specific styles

## Justify why they are needed
Colocation 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4866?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4866** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4866?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4861** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4861?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/PriceCalculatorCmsPage/PriceCalculatorCmsPageContent/PriceCalculatorCmsPageContent.css.ts [hunk 1]
```diff
@@ -27,16 +27,14 @@ export const pageGrid = style({
 })
 
 export const productHeroSection = style([
-  yStack({ justifyContent: 'center' }),
+  sprinkles({ display: 'flex', flexDirection: 'column', justifyContent: 'center' }),
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/PriceCalculatorCmsPage/PriceCalculatorCmsPageContent/PriceCalculatorCmsPageContent.css.ts:30]: What's wrong with `yStack` here? My first thought is it's exactly the same thing, but easier to read

**Review by alebedev — APPROVED** [review]
Nice!

**Review by guilhermespopolin — APPROVED** [review]
👌🏻

File: apps/store/src/features/priceCalculator/PriceCalculatorCmsPage/PriceCalculatorCmsPageContent/PriceCalculatorCmsPageContent.css.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/features/priceCalculator/PriceCalculatorCmsPage/PriceCalculatorCmsPageContent/PriceCalculatorCmsPageContent.css.ts:30]: In this case it's just a single child we want to position with flex. For me a stack would contain multiple children. But we could use `yStack` if you think it makes more sense/easier to read

**Comment by gustaveen** [comment]
### Merge activity

* **Oct 21, 10:11 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4866).

**Pull request merged**